inspirehep/modules/workflows/views/callback.py

```diff
@@ -97,7 +148,13 @@ def webcoll_callback():
 
     Expects the request data to contain a list of record ids in the
     recids field.
+
+    DEPRECATED: replaced by the simpler continue_workflow_callback
```

Comment:
Ok, I'll try to fix then the current callbacks. Do you have any idea why does it use redis? (IMO the queue of pending records should be in a reliable storage, not in cache)